package.json

```
@@ -8,15 +8,19 @@
   "bin": {
     "station": "./bin/station.js"
   },
+  "engines": {
+    "node": "^16.13 || >=18.12"
```

why shouldn't someone on node 18.0 be able to run this? I don't see an advantage in forcing people to upgrade nodejs. Depending on the environment, they might not be able to (corporate policies etc), unnecessary limiting the possible install base of Station.
```suggestion
    "node": ">=16"
```

---

The 16.x versions before 16.13 and 18.x versions before 18.12 were not LTS yet.

In the past, it was necessary to choose a non-zero server-minor version to get access to features added later in the lifecycle of the major version. E.g. top-level await was added in 14.8.

Anyhow, I don't have any particular argument or requirement to reject older 16.x and 18.x, so I am fine to drop the semver-minor part from the version spec.

However: your suggested change adds support for Node.js 17, which is no longer maintained. Are you fine with that? We don't run CI for that version.

---

Also: do we want to automatically support all future Node.js versions by using `>=18`? Maybe the process of adding support for a new major version should be manual - add the new version to the CI matrix first, ensure all tests pass, only then update the version range.